scripts/ci/check-connect-data.ts

```diff
@@ -180,15 +180,15 @@ const updateConfigFromJSON = async () => {
 
             const commitMessage = 'chore(connect): update device authenticity config';
             await exec('git', ['checkout', '-b', branchName]);
-            commit({
+            await commit({
```

Comment:
There was missing await here so, sometimes it was pushing before commiting was completed failing.